plugins/MmsdHelpers/src/View/Helper/BsFormHelper.php

```diff
@@ -369,8 +369,10 @@ public function check(string $name, array $options = [], array $config = [])
     
     // START EXPERIMENTAL
 
-    public function checkGroup(array $fullList, array $data = null, string $modelName, string $foreignKey, string $bitField, array $options = [], array $config = [], string $primaryKey = 'id')
+    public function checkGroup(array $fullList, array $data = null, string $modelName, string $foreignKey, string $bitField, array $options = [], bool $forPrint = null, array $config = [], string $primaryKey = null)
     {
+        $primaryKey = (is_null($primaryKey)) ? 'id' : $primaryKey;
```

Comment:
I'm just curious how this is different from setting the default value of 'id' to $primaryKey in the function signature?

Reply:
> I'm just curious how this is different from setting the default value of 'id' to $primaryKey in the function signature?

There really is no benefit to the way I wrote that, and setting the default $primaryKey val to 'id' in the function signature is the more elegant way, so I'm glad you simply made it a default in the function sig. :)